library-app/src/components/Books/BooksList/BooksList.tsx

```diff
@@ -1,18 +1,62 @@
 import BooksItem from '../BooksItem/BooksItem'
 import styles from './BooksList.module.css'
+import { useEffect, useState } from 'react'
+import Modal from '../../Layout/Modal'
+import ManageBookForm from './ManageBookForm'
+import { getBooksRequest } from '../../../services/BooksServices'
+import Author from '../../../models/author.model'
+
+export interface BookBodyDataGet {
```

Comment:
Why is Author model at the separate model file and this BookBodyDataGet model in tsx file?

Reply:
it is relocated now